charts/templates/secret.yaml

```diff
@@ -1,3 +1,9 @@
+{{- $adminPassword := (randAlphaNum 8) | b64enc | quote }}
```

Comment:
Why is this base64 encoded? Is this best practice or something?

Reply:
Followed the example from https://kubernetes.io/docs/tasks/configmap-secret/managing-secret-using-kubectl/